lib/rubocop/cop/style/invertible_unless_condition.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for usages of `unless` which can be replaced by `if` with inverted condition.
```

Comment:
Probably it's a good idea to mention here that the idea is that code without `unless` is easier to read, but that's subjective that's why the cop is disabled by default.

Reply:
Added.